src/sentry/runner/commands/backup.py

```diff
@@ -299,23 +299,25 @@ def sort_dependencies():
             # Now add a dependency for any FK relation with a model that
             # defines a natural key
             for field in model._meta.fields:
-                if hasattr(field.remote_field, "model"):
-                    rel_model = field.remote_field.model
-                    if rel_model != model:
-                        # TODO(hybrid-cloud): actor refactor.
-                        # Add cludgy conditional preventing walking actor.team_id, actor.user_id
-                        # Which avoids circular imports
-                        if model == Actor and (rel_model == Team or rel_model == User):
-                            continue
+                rel_model = getattr(field.remote_field, "model", None)
+                if rel_model is not None and rel_model != model:
+                    # TODO(hybrid-cloud): actor refactor.
+                    # Add cludgy conditional preventing walking actor.team_id, actor.user_id
+                    # Which avoids circular imports
+                    if model == Actor and (rel_model == Team or rel_model == User):
+                        continue
 
-                        deps.append(rel_model)
+                    deps.append(rel_model)
 
             # Also add a dependency for any simple M2M relation with a model
             # that defines a natural key.  M2M relations with explicit through
             # models don't count as dependencies.
-            for field in model._meta.many_to_many:
-                rel_model = field.remote_field.model
-                if rel_model != model:
+            many_to_many_fields: list[ManyToManyField] = [
```

Comment:
the annotation shouldn't be needed here because the `if` is narrowing

Reply:
Done.